src/components/Outline.js

```diff
@@ -50,7 +64,7 @@ class Outline extends Component {
 
     return functions
       .filter(func => func.value != "anonymous")
-      .map(this.renderFunction);
+      .map(func => this.renderFunction(func));
```

Comment:
yes, it binds the `this` value. I just took a look through a few of the other files though and I noticed that doing the binding in the constructor is more common... that be better?

ie)
```
constructor(props) {
  super(props);
  this.renderFunction = this.renderFunction.bind(this)
}
```

both achieve the same result, as far as I know anyway. Happy to change it!